packages/melmatcheq.lv2/PKGBUILD

```
@@ -0,0 +1,48 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_name=MelMatchEQ.lv2
+pkgname=${_name,,}
+pkgver=0.1
+pkgrel=1
+pkgdesc='A profiling EQ using a 26 step Mel Frequency Band LV2 plugin'
+arch=(aarch64 x86_64)
+url="https://github.com/brummer10/$_name"
+license=(GPL2)
+groups=(lv2-plugins pro-audio)
+depends=(cairo glibc gcc-libs libx11)
+makedepends=(lv2)
+checkdepends=(lv2lint)
+optdepends=('lv2-host: for loading the LV2 plugin')
+source=("$_name-$pkgver.tar.gz::https://github.com/brummer10/$_name/archive/refs/tags/v$pkgver.tar.gz"
+        'fix-shortcut.patch::https://patch-diff.githubusercontent.com/raw/brummer10/MelMatchEQ.lv2/pull/2.diff'
+        'fix-manifest.patch::https://github.com/brummer10/MelMatchEQ.lv2/pull/4.diff')
```

I know we're using both formats throughout this repo, but let's keep it consistent here:
```suggestion
        'fix-shortcut.patch::https://github.com/brummer10/MelMatchEQ.lv2/pull/2.patch'
        'fix-manifest.patch::https://github.com/brummer10/MelMatchEQ.lv2/pull/4.patch')
```
(needs updating the sha256 sums as well because here patch != diff)